### PR TITLE
239 Contributing docs

### LINKED ISSUE #239: Create a CONTRIBUTING.md doc
## What problem does this address?

New contributors should have a clear outline of how we work, and how to get involved.

## What is your proposed solution?

Create a CONTRIBUTING.md document and update the readme to reference it.

### PR DESCRIPTION
**Pull request by Sephsekla**
## What?

Add a `CONTRIBUTING.md` file with instructions for contributors.

## Why?

Resolves #239

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -2,53 +2,18 @@
 
 > A feature plugin for WordPress, which aims to create a new (better) way to manage and deliver notifications to the relevant audience.
 
-- Contributors: schlessera, psykro, raaaahman, danbilauca, sephsekla
+- Contributors: schlessera, psykro, raaaahman, danbilauca, Sephsekla, erikyo, JasonTheAdams, johnhooks
```
Comment (johnhooks) [inline README.md:5]: ❤️ thanks!

**Comment by johnhooks** [comment]
I think the initial, "how do you get connected" section is still very effective. 

It would be helpful to define a roadmap and explain where the current focus is. This probability doesn't belong in the CONTRIBUTING.md, but linking to where to find that information could direct new contributors towards the areas we need help (possibly linking to a project board). The issues list, by its nature, is rather unorganized and hard to look at as a newcomer and make any sense of the priorities.

List the current maintainers and who has the authority to merge pull requests. 

What are the criteria for accepting pull requests:

- bugs: merged as quickly as possible
- refactors: merged if deemed necessary by the maintainers
- features: first considered whether or not it fits the project, if determined to fit with the projects goals, it’s placed into the project board, into the system and when complete is merged when it fits with the release schedule.

I'm pretty new to this, so those are just ideas. It can be hard to tell sometimes in projects like this whether or not contributions are welcomed. Providing a "when contributing" section that lays out how someone should approach offering a new idea can give them confidence we want their input and have thought through the process.

**Comment by Sephsekla** [comment]
@johnhooks there's still a lot that could be added here, but for now I'd like to get something merged and then we can get 0.2.0 out. What do you think?

**Review by johnhooks — CHANGES_REQUESTED** [review]
This looks great @Sephsekla, I think you did a great job explaining the process of the PR workflow. Only suggestion, could you add the `CONTRIBUTING.md` to the `.distignore` file?

**Comment by Sephsekla** [comment]
I don't think it actually makes a difference to the release process now we're using wp-scripts, but it's at the very least complete.

**Review by johnhooks — APPROVED** [review]
LGTM!

**Comment by johnhooks** [comment]
> I don't think it actually makes a difference to the release process now we're using wp-scripts, but it's at the very least complete.

@Sephsekla the only reason I thought to add it is because we have that file and might use it someday, we should either keep it up to date or remove it.

**Pull request merged**